src/searchModal.js

```diff
@@ -258,15 +258,50 @@ export default function searchModalFactory(config) {
         });
 
         /**
-         * clicking on class filter input will toggle resource selector,
-         * will preventDefault to avoid focus on input field,
-         * and will stopPropagation to prevent be closed
+         * Pressing space, enter, esc, backspace 
+         * on class filter input will toggle resource selector
+         */
+        $classFilterInput.on('keyup', e => {
+            const spacePressed = e.code === "Space" || e.which === 32;
```

Comment:
Please, check in tao-core-ui or tao-core-sdk some util/helper for key codes. If we have such helper, please, use it.

Reply:
Yes, there is a method for that in _tao-core-sdk-fe/src/util/shortcut/registry.js:147_
But as I see, it is not used in many places in tao-core-ui-fe 
<img width="446" alt="Screenshot" src="https://user-images.githubusercontent.com/11773029/105164119-ce6b7b00-5b25-11eb-8971-2fe071c5752c.png">

Should I create tech debt issue in Jira for refactoring this in other places?

Reply:
Please check my refactoring